hudi-flink/src/main/java/org/apache/hudi/sink/compact/FlinkCompactionConfig.java

```diff
@@ -83,6 +83,12 @@
   @Parameter(names = {"--compaction-max-memory"}, description = "Max memory in MB for compaction spillable map, default 100MB.", required = false)
   public Integer compactionMaxMemory = 100;
 
+  @Parameter(names = {"--compaction-target-io"}, description = "Target IO per compaction (both read and write) for batching compaction, default 512000M.", required = false)
```

Comment:
Should FlinkOptions#COMPACTION_TARGET_IO default value change to 512000 to align with HoodieCompactionConfig#DEFAULT_TARGET_IO_PER_COMPACTION_IN_MB and here?

Reply:
> Should FlinkOptions#COMPACTION_TARGET_IO default value change to 512000 to align with HoodieCompactionConfig#DEFAULT_TARGET_IO_PER_COMPACTION_IN_MB and here?

After we used a large amount of data to test writing, we found that the online compaction setting FlinkOptions#COMPACTION_TARGET_IO  cannot be setted too large, which would seriously affect the running of the writing tasks. In contrast, the offline compaction target io default value can be set larger while we have enough memory.